### ISSUE TITLE
Scanner crashes sometimes when taking a photo. Terminated due to memory issue

### ISSUE THREAD

**Issue by julianklumpers** [Michaelvilleneuve/react-native-document-scanner#57]
So now and then the scanner crashed right after taking a photo. the log in xcode says the following: Terminated due to memory issue.

Someone else experiencing this problem or maybe know how to solve this.

Thanks.

**Comment by mariot** [comment]
I had that for old phones running Android.
You could try to ask for more memory with `android:largeHeap="true"`

https://stackoverflow.com/questions/27396892/what-are-advantages-of-setting-largeheap-to-true

**Comment by Michaelvilleneuve** [comment]
Mariot this issue is iOS specific apparently

**Comment by humphreyja** [comment]
Yeah I ran into this issue as well. The iOS code is really old and filled with memory leaks. I actually rebuilt this package from the ground up to fix these issues and update the iOS stuff. I changed the API so be sure to check out the full example for how to use it. Or just take the iOS code and add a PR to this package to fix the issues. 

https://github.com/HarvestProfit/react-native-rectangle-scanner
